README.md

```diff
@@ -1,13 +1,8 @@
-# Project Name
+# Guess Who game
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+This project of a Guess Who game, was made as a Week 5 project for Technigo bootcamp.
+The aim was to create a classic board game "Guess Who" and practice creating objects, arrays and connecting it to the DOM.
 
```

Comment:
It would be amazing to read a bit more about your process here, what went well and/or not and how it was solved :)

Reply:
Definitely! Make it shine because it is an invitation to look inside your code 🎇